tests/bson/bson-utcdatetime-serialization_error-002.phpt

```diff
@@ -6,7 +6,7 @@ MongoDB\BSON\UTCDateTime unserialization requires "milliseconds" string to parse
 require_once __DIR__ . '/../utils/tools.php';
 
 echo throws(function() {
-    unserialize('C:24:"MongoDB\BSON\UTCDateTime":40:{a:1:{s:12:"milliseconds";s:7:"INVALID";}}');
+    unserialize('C:24:"MongoDB\BSON\UTCDateTime":42:{a:1:{s:12:"milliseconds";s:9:"1234.5678";}}');
```

Comment:
I would have kept  the length of the argument the same, as to not have to mess with lengths in hardcoded serialized strings. But that's done now.